src/test/java/org/jitsi/meet/test/web/WebParticipantFactory.java

```diff
@@ -322,4 +350,24 @@ private URL getRemoteDriverAddress()
 
         return null;
     }
+
+    /**
+     * Downloads extension from chrome webstore.
+     * @param id - the id of the extension.
+     * @returns <tt>File</tt> associated with the downloaded extension.
+     */
+    private static File downloadExtension(String id)
+        throws IOException
+    {
+        URL website = new URL(
+            EXT_DOWNLOAD_URL_PREFIX + id + EXT_DOWNLOAD_URL_SUFIX);
```

Comment:
why not user string format here ?

Reply:
How? My Java is a little rusty... What will be the benefit?

Reply:
https://dzone.com/articles/java-string-format-examples

It is weird that there are two constants for something which is actually one thing, where prefix is never used without the suffix.

It's possible that the % signs in the string would have to be escaped.